machines/openstack/worker.machineset.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
We are planning to move the machineSet specs generation into the installer, so may be move this into that repo, so then we just make sure we deploy the right machine controller here